src/rgw/rgw_op.cc

```diff
@@ -1828,6 +1896,10 @@ void RGWStatBucket::execute()
       op_ret = -EINVAL;
     }
   }
+  map<string, bufferlist>::iterator cmp = s->bucket_attrs.find(RGW_ATTR_COMPRESSION);
+  if (cmp != s->bucket_attrs.end()) {
+    ::decode(bucket.size, cmp->second);
+  }
```

Comment:
could you please add a `struct RGWBucketCompressionInfo` or similar with versioned encode/decode in case we need to extend it later on?